codalab/client/local_bundle_client.py

```diff
@@ -167,10 +167,10 @@ def get_bundle_target(self, target):
         return (self.model.get_bundle(bundle_uuid), subpath)
 
     def get_worksheet_uuid(self, base_worksheet_uuid, worksheet_spec):
-        if worksheet_spec == '' or worksheet_spec == worksheet_util.HOME_WORKSHEET:
-            # Default worksheet name: take the username.
-            worksheet_spec = spec_util.home_worksheet(self._current_user_name())
-            return self.new_worksheet(worksheet_spec, True)
+        if worksheet_spec == '' or worksheet_spec == worksheet_util.DASHBOARD:         
```

Comment:
Just wanna clarify. Do we want our landing page to be the dashboard or the home worksheet? I think I personally prefer the former.